src/app/client/src/app/modules/learn/components/learn-page/learn-page.component.ts

```diff
@@ -386,6 +386,9 @@ export class LearnPageComponent implements OnInit, OnDestroy, AfterViewInit {
         const formatedContent = this.utilService.processContent(content, constantData, dynamicFields, metaData);
         formatedContent.metaData.mimeType = 'application/vnd.ekstep.content-collection'; // to route to course page
         formatedContent.metaData.contentType = 'Course'; // to route to course page
+        if (content.content.trackable) {
```

Comment:
why is the content.content.trackable ?

Reply:
Because of _.map(this.enrolledCourses, **content** => { })  
enrolledCourses has content object which has content details that's why it is content.content.trackable